data/prrajput1199.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Pratik Rajput",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open-Source enthusiastic",
+  "avatar": "https://github.com/prrajput1199.png"
```

Comment:
Hi, you forget to add tags
example:
  "tags": ["Open Source", "React"],

![image](https://user-images.githubusercontent.com/88339569/215154321-056b0b18-022a-436f-86b3-688a8083d12d.png)

Reply:
@devvsakib can you tell me where I should add tags? I am unable to understand.

Reply:
@prrajput1199  sure, 
this way you can add your tags
![image](https://user-images.githubusercontent.com/88339569/215156434-277aac27-655b-458e-9bc9-82e4f50f3d74.png)

Reply:
@devvsakib Can you check now?? I have made the changes